ui/pages/first-time-flow/create-password/new-account/new-account.component.js

```diff
@@ -161,40 +176,74 @@ export default class NewAccount extends PureComponent {
             }}
             href="#"
           >
-            {`< ${t('back')}`}
+            <BackIcon className="mr-2" />{t('back')}
           </a>
         </div>
         <div className="first-time-flow__header">{t('createPassword')}</div>
         <form className="first-time-flow__form" onSubmit={this.handleCreate}>
-          <TextField
-            id="create-password"
-            label={t('newPassword')}
-            type="password"
-            className="first-time-flow__input"
-            value={password}
-            onChange={(event) => this.handlePasswordChange(event.target.value)}
-            error={passwordError}
-            autoFocus
-            autoComplete="new-password"
-            margin="normal"
-            fullWidth
-            largeLabel
-          />
-          <TextField
-            id="confirm-password"
-            label={t('confirmPassword')}
-            type="password"
-            className="first-time-flow__input"
-            value={confirmPassword}
-            onChange={(event) =>
-              this.handleConfirmPasswordChange(event.target.value)
-            }
-            error={confirmPasswordError}
-            autoComplete="confirm-password"
-            margin="normal"
-            fullWidth
-            largeLabel
-          />
+          <div className='first-time-flow__inputWrap'>
```

Comment:
```suggestion
          <div className='first-time-flow__input-wrap'>
```
- To be consistent with the coding style

Reply:
fixed in 61b20a179b653d94185890b9069c56a5f6a9dd34